APITesters/SwiftAPITester/SwiftAPITester/StoreProductDiscountAPI.swift

```diff
@@ -36,3 +36,16 @@ func checkPaymentModeEnum() {
     @unknown default: fatalError()
     }
 }
+
+var type: StoreProductDiscount.DiscountType!
+
+func checkTypeEnum() {
```

Comment:
I know we don't use this approach everywhere, but you could pass the enum as a parameter to avoid the !

Reply:
I tried this but it ended up making this API super weird 👇 

![Screen Shot 2022-02-09 at 7 08 58 PM](https://user-images.githubusercontent.com/401294/153317470-262bcfe9-4d58-4fc2-a376-91db89b2a532.png)

Reply:
Oh right. Technically we don't have to call the methods but you can leave it as is for consistency :)